runtime/near-vm-logic/src/config.rs

```diff
@@ -541,4 +577,62 @@ impl ExtCosts {
             validator_total_stake_base => config.validator_total_stake_base,
         }
     }
+
+    pub fn count() -> usize {
```

Comment:
Let's try to make it `const`: https://doc.rust-lang.org/reference/items/functions.html#const-functions